### DISCUSSION TITLE
WinError 10051?

### DISCUSSION THREAD

**Discussion by luu5** [kubilus1/autoortho#39] [General]
Just installed and started the XP11. Cache is filling nicely but in the logs I see quite a many of these lines, are they an issue?

Failed getting: Chunk(37463,17674,BI,16,16) () {}, re-submit.
Failed getting: Chunk(36993,17600,BI,16,16) () {}, re-submit.
Failed getting: Chunk(37458,17674,BI,16,16) () {}, re-submit.
Failed to get chunk Chunk(37460,17667,BI,16,16) on server b. Err: <urlopen error [WinError 10051] A socket operation was attempted to an unreachable network>
Failed to get chunk Chunk(37457,17666,BI,16,16) on server b. Err: <urlopen error [WinError 10051] A socket operation was attempted to an unreachable network>
Failed to get chunk Chunk(37466,17669,BI,16,16) on server a. Err: <urlopen error [WinError 10051] A socket operation was attempted to an unreachable network>
Failed getting: Chunk(37460,17667,BI,16,16) () {}, re-submit.
Failed getting: Chunk(37457,17666,BI,16,16) () {}, re-submit.
Failed getting: Chunk(37466,17669,BI,16,16) () {}, re-submit.
Failed to get chunk Chunk(37461,17674,BI,16,16) on server d. Err: <urlopen error [WinError 10051] A socket operation was attempted to an unreachable network>
Failed getting: Chunk(37461,17674,BI,16,16) () {}, re-submit.
Failed to get chunk Chunk(37465,17669,BI,16,16) on server d. Err: <urlopen error [WinError 10051] A socket operation was attempted to an unreachable network>
Failed getting: Chunk(37465,17669,BI,16,16) () {}, re-submit.
Failed to get chunk Chunk(37460,17674,BI,16,16) on server a. Err: <urlopen error [WinError 10051] A socket operation was attempted to an unreachable network>
Failed to get chunk Chunk(37459,17674,BI,16,16) on server b. Err: <urlopen error [WinError 10051] A socket operation was attempted to an unreachable network>
Failed getting: Chunk(37460,17674,BI,16,16) () {}, re-submit.
Failed getting: Chunk(37459,17674,BI,16,16) () {}, re-submit.
Failed to get chunk Chunk(37063,17840,BI,16,16) on server c. Err: <urlopen error [WinError 10051] A socket operation was attempted to an unreachable network>
Failed getting: Chunk(37063,17840,BI,16,16) () {}, re-submit.

**Comment by luu5** [comment]
I guess this also makes the start to take up a long time... Now it has been starting for 35 minutes.

**Comment by kubilus1** [comment]
This just looks like a windows networking error of some sort.
